cas-server-support-ldap/src/main/java/org/jasig/cas/authentication/LdapAuthenticationHandler.java

```diff
@@ -152,7 +153,7 @@ protected HandlerResult authenticateUsernamePasswordInternal(final UsernamePassw
         final AuthenticationResponse response;
         try {
             logger.debug("Attempting LDAP authentication for {}", upc);
-            final String password = getPasswordEncoder().encode(upc.getPassword());
+            final String password = Strings.nullToEmpty(getPasswordEncoder().encode(upc.getPassword()));
```

Comment:
`DefaultPasswordEncoder` will return null if the password is already null or you haven't defined an encodingAlgorithm